lms/static/sass/course/instructor/_writable_gradebook.scss

```diff
@@ -0,0 +1,105 @@
+@mixin table-filter-input {
+    border: 1px $gray solid;
+    background-color: $white;
+    border-radius: 4px;
+}
+
+.gradebook-content {
+
+    .dataTables_filter {
+        input {
+            @include table-filter-input;
+        }
+    } 
+
+    .gradebook-filter {
+        select {
+            @include table-filter-input;
+        }
+    }
+
+    .view-container {
+        padding-bottom: 30px;
+        span {
```

Comment:
NIT: I'm not sure if there is a way to assign classes to things, but in general, styling element selectors isn't a good practice. Now every span, even if it gets a class added to it, will have margin-right of 5px when in `.gradebook-content > .view-container`